src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandResultBuilder.java

```diff
@@ -132,8 +155,8 @@ public CommandResult build() {
           case SINGLE_DOCUMENT ->
               documents.isEmpty()
                   ? null
```

Comment:
Is this the correct response when documents are empty?

So for the previous collection path, if no documents, it will be
{
    "data": {
        "document": null
    }
}
But now, it will just be
{}.

Before this PR, table path is already this way, {}.
Wonder if this is intentional or?

```
          case SINGLE_DOCUMENT ->
              documents.isEmpty()
                  ?  new CommandResult.SingleResponseData(null)
                  : new CommandResult.SingleResponseData(documents.getFirst());
```

Reply:
fixed below, and does not return responseData if there is any errors. All integration tests updated to catch this bug.